examples/tic-tac-toe/tic-tac-toe.js

```diff
@@ -51,6 +53,11 @@ export const TicTacToe = Game({
       }
 
       return { ...G, cells };
+    },
+    resetGame(G) {
+      const cells = noFill(9);
+
+      return { ...G, cells }
```

Comment:
Need to also reset `log` here.

Reply:
Ok

Reply:
the `log` list has not any reference inside `moves`